src/styles/index.scss

```diff
@@ -1,3 +1,114 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+.product-card {
+  width: 200px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)